pkgs/top-level/php-packages.nix

```diff
@@ -49,12 +49,19 @@
 }:
 
 lib.makeScope pkgs.newScope (self: with self; {
-  buildPecl = import ../build-support/build-pecl.nix {
+  buildPecl = import ../build-support/php/build-pecl.nix {
     php = php.unwrapped;
     inherit lib;
     inherit (pkgs) stdenv autoreconfHook fetchurl re2c nix-update-script;
   };
 
+  buildPHPproject = import ../build-support/php/build-php.nix {
+    php = php.unwrapped;
```

Comment:
Why unwrapped?

Reply:
I just used what was existing in `build-pecl`

Reply:
I removed it.